package.json

```diff
@@ -286,6 +286,7 @@
     "@azure/arm-cosmosdb": "13.0.0",
     "@azure/arm-monitor": "6.1.1",
     "@azure/arm-resourcegraph": "4.2.1",
+    "ads-extension-telemetry": "1.0.0",
```

Comment:
You should use `@microsoft/ads-extension-telemetry` https://www.npmjs.com/package/@microsoft/ads-extension-telemetry